Explain to the build system that we would like to use MacPorts versions
of these libraries, not the bundled ones which may become out of date.
--- code/CMakeLists.txt.orig
+++ code/CMakeLists.txt
@@ -1115,6 +1115,5 @@
 # pugixml
-IF(ASSIMP_HUNTER_ENABLED)
-  hunter_add_package(pugixml)
+IF(TRUE)
   find_package(pugixml CONFIG REQUIRED)
 ELSEIF(NOT TARGET pugixml::pugixml)
   SET( Pugixml_SRCS
@@ -1170,7 +1170,6 @@
 # minizip/unzip
-IF(ASSIMP_HUNTER_ENABLED)
-  hunter_add_package(minizip)
-  find_package(minizip CONFIG REQUIRED)
+IF(TRUE)
+  # minizip is linked via pkg-config in the top-level CMakeLists.txt
 ELSE()
   SET( unzip_SRCS
     ../contrib/unzip/crypt.h
--- cmake-modules/assimp-plain-config.cmake.in.orig
+++ cmake-modules/assimp-plain-config.cmake.in
@@ -1 +1,2 @@
-@PACKAGE_INIT@
+@PACKAGE_INIT@
+find_package(pugixml CONFIG REQUIRED)
